wandb/sdk/wandb_init.py

```diff
@@ -642,6 +642,9 @@ def init(self) -> Union[Run, RunDisabled, None]:  # noqa: C901
             tel.cli_version = wandb.__version__
             tel.python_version = platform.python_version()
             tel.platform = f"{platform.system()}-{platform.machine()}".lower()
+            launch_run_queue_item_id = wandb.env.get_launch_run_queue_item_id()
+            if launch_run_queue_item_id:
+                tel.launch_run_queue_item_id = launch_run_queue_item_id
```

Comment:
yep, again why do you need to have it in telemetry?
maybe it makes more sense to add an extra message in the run's construction? basically after you init the run in the launch you will do an extra step of adding the queue id or whatever you need to do... i guess i don't know exactly how you spin up new runs, but that sounds like a better place than in a general run initialization

Reply:
it was purely convenience, im happy to rewire it :)

Reply:
@kptkin where would that extra message live? in `wandb_run`?

Reply:
yeah, i don't think telemetry is the right place even if you decide to keep it in the config...